lib/reporter.js

```diff
@@ -225,6 +226,13 @@ class CucumberReporter {
         })
     }
 
+    getTitle (featureOrScenario) {
+        const name = featureOrScenario.getName()
+        const tags = featureOrScenario.getTags ? featureOrScenario.getTags() : []
```

Comment:
> Validation for safety reason

I would rather have it break the code than doing that. When you are sure `getTags` is available than there should be no validation.